.gitignore

```diff
@@ -65,11 +65,10 @@ test-results/
 
 # MMI Playwright
 public/playwright
-test/e2e/mmi/.env
-test/e2e/mmi/playwright.config.ts
-test/e2e/mmi/specs/**/*-darwin.png
-test/e2e/mmi/dist/
-
+test/e2e/playwright/.env
```

Comment:
shouldn't it be `test/e2e/playwright/mmi/.env`?

Reply:
Yes that's correct. I have fixed it